include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -88,8 +88,8 @@ def TTG_CmpFOp : TTG_Op<"cmpf", [NoSideEffect, Elementwise,
 }
 
 // TODO: migrate to arith::SelectOp on LLVM16
-def TTG_SelectOp : TTG_Op<"select", [NoSideEffect, Elementwise, 
-                                     SameOperandsAndResultShape, 
+def TTG_SelectOp : TTG_Op<"select", [NoSideEffect, Elementwise,
```

Comment:
It's a little tricky because the Arith dialect's verifier doesn't always verify that encodings of the operands are the same. Our custom op allows us to do that